Erreur création utilisateur : Error
    at Query.run (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/sequelize.js:315:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresQueryInterface.select (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)
    at async User.findAll (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/model.js:1140:21)
    at async User.findOne (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/model.js:1240:12)
    at async /home/mateo/Documents/Code/aboWatch/backend/routes/userRoutes.js:44:30 {
  name: 'SequelizeDatabaseError',
  parent: error: column "password" does not exist
      at Parser.parseErrorMessage (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:517:28)
      at addChunk (node:internal/streams/readable:368:12)
      at readableAddChunk (node:internal/streams/readable:341:9)
      at Readable.push (node:internal/streams/readable:278:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 108,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '38',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '3665',
    routine: 'errorMissingColumn',
    sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
    parameters: undefined
  },
  original: error: column "password" does not exist
      at Parser.parseErrorMessage (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:517:28)
      at addChunk (node:internal/streams/readable:368:12)
      at readableAddChunk (node:internal/streams/readable:341:9)
      at Readable.push (node:internal/streams/readable:278:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 108,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '38',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '3665',
    routine: 'errorMissingColumn',
    sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
    parameters: undefined
  },
  sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
  parameters: {}
}
Erreur création utilisateur : Error
    at Query.run (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/dialects/postgres/query.js:50:25)
    at /home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/sequelize.js:315:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async PostgresQueryInterface.select (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/dialects/abstract/query-interface.js:407:12)
    at async User.findAll (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/model.js:1140:21)
    at async User.findOne (/home/mateo/Documents/Code/aboWatch/node_modules/sequelize/lib/model.js:1240:12)
    at async /home/mateo/Documents/Code/aboWatch/backend/routes/userRoutes.js:44:30 {
  name: 'SequelizeDatabaseError',
  parent: error: column "password" does not exist
      at Parser.parseErrorMessage (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:517:28)
      at addChunk (node:internal/streams/readable:368:12)
      at readableAddChunk (node:internal/streams/readable:341:9)
      at Readable.push (node:internal/streams/readable:278:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 108,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '38',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '3665',
    routine: 'errorMissingColumn',
    sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
    parameters: undefined
  },
  original: error: column "password" does not exist
      at Parser.parseErrorMessage (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:285:98)
      at Parser.handlePacket (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:122:29)
      at Parser.parse (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/parser.js:35:38)
      at Socket.<anonymous> (/home/mateo/Documents/Code/aboWatch/node_modules/pg-protocol/dist/index.js:11:42)
      at Socket.emit (node:events:517:28)
      at addChunk (node:internal/streams/readable:368:12)
      at readableAddChunk (node:internal/streams/readable:341:9)
      at Readable.push (node:internal/streams/readable:278:10)
      at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
    length: 108,
    severity: 'ERROR',
    code: '42703',
    detail: undefined,
    hint: undefined,
    position: '38',
    internalPosition: undefined,
    internalQuery: undefined,
    where: undefined,
    schema: undefined,
    table: undefined,
    column: undefined,
    dataType: undefined,
    constraint: undefined,
    file: 'parse_relation.c',
    line: '3665',
    routine: 'errorMissingColumn',
    sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
    parameters: undefined
  },
  sql: `SELECT "userid" AS "id", "username", "password" FROM "user" AS "User" WHERE "User"."username" = 'APIPIII';`,
  parameters: {}
}
